crates/turborepo-repository/src/package_manager/mod.rs

```diff
@@ -351,6 +355,18 @@ impl PackageManager {
         Ok((inclusions, exclusions))
     }
 
+    pub fn workspace_glob_source(&self, root_path: &AbsoluteSystemPath) -> AbsoluteSystemPathBuf {
+        match self {
+            PackageManager::Pnpm | PackageManager::Pnpm6 => {
+                root_path.join_component("pnpm-workspace.yaml")
+            }
+            PackageManager::Berry
+            | PackageManager::Yarn
+            | PackageManager::Npm
+            | PackageManager::Bun => root_path.join_component("package.json"),
+        }
```

Comment:
This is partially duplicated code from `workspace_configuration_path`
```suggestion
       root_path.join_component(self.workspace_configuration_path().unwrap_or("package.json")
```